js/samples/04.ai.b.messageExtensions.AI-ME/README.md

```diff
@@ -111,6 +109,12 @@ app.messageExtensions.submitAction<SubmitData>('CreatePost', async (context: Tur
     cd teams-ai/js/samples/04.ai.b.messageExtensions.aime/
     ```
 
+4. Duplicate the `sample.env` in the `teams-ai/js/samples/04.ai.b.messageExtensions.AI-ME` folder. Rename the file to `.env`. 
+
+5. If you are using OpenAI then only keep the `OPENAI_KEY` and add in your key. Otherwise if you are using AzureOpenAI then only keep the `AZURE_OPENAI_KEY`, `AZURE_OPENAI_ENDPOINT`variables and fill them in appropriately.
+
```

Comment:
The purpose of some of Steve's refactoring is so that calls can be made to any assistant -- could you modify the wording so that they can do AOAI OR OAI, OR they can do both?

Reply:
I think we should stick to just AOAI or OAI, not to confuse and suggest the use of both to developers (this would be tricky during debugging).